community-website/package.json

```diff
@@ -4,7 +4,7 @@
   "version": "1.0.0",
   "scripts": {
     "start": "NODE_ENV=development node src/community-project-boilerplate-docgen/start.js",
-    "docs:build": "NODE_ENV=production node src/community-project-boilerplate-docgen/build.js",
+    "docs:build": "NODE_ENV=production node src/community-project-boilerplate-docgen/build.js && cp _redirects docs",
```

Comment:
I think maybe it's best that the redirection is functional only when we decide to deprecate V2, no?

Reply:
It seems fine to me, but you're right, we can wait to redirect the root until v3 is stable?